TractorMod/Framework/Attachments/ScytheAttachment.cs

```diff
@@ -46,27 +84,46 @@ public override bool Apply(Vector2 tile, SObject tileObj, TerrainFeature tileFea
                 if (dirt.crop == null)
                     return false;
 
-                if (dirt.crop.dead)
+                if (dirt.crop.dead && this.ClearDeadCrops)
                 {
                     this.UseToolOnTile(new Pickaxe(), tile); // clear dead crop
                     return true;
                 }
+
+                List<int> flowers = new List<int> { 376, 591, 593, 595, 597, 421 };
                 if (dirt.crop.harvestMethod == Crop.sickleHarvest)
+                {
+                    if (flowers.Contains(dirt.crop.indexOfHarvest))
+                        return this.HarvestFlowers && dirt.performToolAction(tool, 0, tile, location);
+                    else
+                        return this.HarvestCrops && dirt.performToolAction(tool, 0, tile, location);
+                }
+                else
+                    this.CheckTileAction(location, tile, player);
+
+                /*
```

Comment:
No need to comment out code, since it's all in source control. Feel free to just remove old code.

Reply:
Huh, I thought I removed it once I was finished, I guess I missed that section though.